common/websocket/index.ts

```diff
@@ -41,52 +42,79 @@ class WebSocketService {
         return this.connectedClients;
     }
 
-    private addClient(id: string, connection: WebSocket) {
-        this.connectedClients[id] = connection;
+    private addClient(userId: string, connection: WebSocket) {
+        (this.connectedClients[userId] = this.connectedClients[userId] || []).push(connection);
     }
 
     private removeClient(id: string) {
```

Comment:
you can simplify the method when you store the userId on ws object

Reply:
Done 🙂